### PR TITLE
Home health services

### PR DESCRIPTION
**Pull request by hkong**
this adds/updates some resources for the Home Health Services Order rule set.  It goes with https://github.com/HL7-DaVinci/CDS-Library/pull/48 for testing that ruleset.

### CODE REVIEW

File: fhirResourcesToLoad/o2billy_04_service-request-homehealthservices-G0180.json [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
-            "reference": "Practitioner/pra1255"
-        }
-    ],  
-    "insurance": [
-        {
-          "reference": "Coverage/cov016"
-        }
+  "resourceType": "ServiceRequest",
+  "id": "servreq-g0180-1",
+  "status": "draft",
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2billy_04_service-request-homehealthservices-G0180.json:4]:
```suggestion
  "status": "draft",
  "profile": [
    "http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-servicerequest-r4"
  ],
```

Comment (yunwwang) [inline fhirResourcesToLoad/o2billy_04_service-request-homehealthservices-G0180.json:4]: 1) Add   "profile": [
    "http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-servicerequest-r4"
  ],
2) Need ServiceRequest.basedOn

File: fhirResourcesToLoad/o2teddy_devicerequest_bgm.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+    "resourceType": "DeviceRequest",
+    "id": "pat014-bgm",
+    "meta": {
+        "profile": [
+            "http://hl7.org/fhir/us/davinci-crd/R4/StructureDefinition/profile-devicerequest-r4"
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2teddy_devicerequest_bgm.json:6]:
```suggestion
            "http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-devicerequest-r4"
```

File: fhirResourcesToLoad/o2teddy_devicerequest_bgm.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        ]
+    },
+    "subject": {
+        "reference": "Patient/pat014"
+    },
+    "performer": {
+        "reference": "Practitioner/pra1255"
+    },
+    "reasonCode": [
+        {}
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2teddy_devicerequest_bgm.json:39]: This is invalid JSON

File: fhirResourcesToLoad/o2teddy_devicerequest_bgm.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{
+    "resourceType": "DeviceRequest",
+    "id": "pat014-bgm",
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2teddy_devicerequest_bgm.json:3]: DeviceRequest.authoredOn is required

File: fhirResourcesToLoad/o2teddy_devicerequest_cbgm.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
+{
+    "resourceType": "DeviceRequest",
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2teddy_devicerequest_cbgm.json:2]: DeviceRequest.authoredOn is required

File: fhirResourcesToLoad/o2teddy_devicerequest_cbgm.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    "id": "pat014-cbgm",
+    "meta": {
+      "profile": [
+        "http://hl7.org/fhir/us/davinci-crd/R4/StructureDefinition/profile-devicerequest-r4"
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2teddy_devicerequest_cbgm.json:6]:
```suggestion
        "http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-devicerequest-r4"
```

File: fhirResourcesToLoad/o2teddy_medicationstatement_insulin.json [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    "informationSource": {
+        "reference": "Patient/pat014",
+        "display": "Roosevelt Theodore"
+    },
+    "reasonCode": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "1543202",
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2teddy_medicationstatement_insulin.json:38]: This is not a valid code. Correct code is "111552007"

**Comment by hkong** [comment]
@yunwwang I'm so sorry about forgetting to close this PR earlier.  The proper set of ehr data for Home Health Services is now https://github.com/HL7-DaVinci/test-ehr/pull/39

**Comment by hkong** [comment]
superceded by https://github.com/HL7-DaVinci/test-ehr/pull/39

**Pull request closed without merging**